tools/system_libs.py

```diff
@@ -249,7 +249,7 @@ def get_default_variation(cls, **kwargs):
 
   # A list of flags to pass to emcc.
   # The flags for the parent class is automatically inherited.
-  cflags = ['-Werror']
+  cflags = ['-Werror', '-fno-unroll-loops']
```

Comment:
please add a comment explaining why this is disabled, and maybe with a TODO to look into the possible perf benefits (which might justify it? unlikely though)

Reply:
Whoops, auto-merge again. Will do a follow-up.